cmd/ci-operator/main.go

```diff
@@ -534,6 +537,31 @@ func (o *options) initializeNamespace() error {
 		break
 	}
 
+	if o.givePrAuthorAccessToNamespace {
+		// Generate rolebinding for the PR Author.
+		if len(o.jobSpec.Refs.Pulls) == 1 {
+			rbacClient, err := rbacclientset.NewForConfig(o.clusterConfig)
```

Comment:
More of a question than a review: would it be worth it to create the RBAC
client only once when both `--give-pr-author-access-to-namespace` and
`--delete-when-idle` are set?

Reply:
Good follow up, let's do this. Only call the `Create()` in the loop

Reply:
@droslean